Resources/Maps/_LostParadise/LPPKerberos.yml

```
@@ -18743,8 +18743,8 @@ entities:
         - volume: 2500
           temperature: 234.99979
           moles:
-          - 21.824879
-          - 82.10312
+          - 20.078888
```

yamllint

[error] wrong indentation: expected 12 but found 10

(indentation)